x-pack/plugins/apm/public/components/app/service_map/popover/anomaly_detection.tsx

```diff
@@ -65,6 +65,7 @@ export function AnomalyDetection({ serviceName, serviceAnomalyStats }: Props) {
   const severity = getSeverity(anomalyScore);
   const actualValue = serviceAnomalyStats?.actualValue;
   const mlJobId = serviceAnomalyStats?.jobId;
+  // TODO check the type
```

Comment:
Should you check it?

Reply:
that's a note for me to check what other cases we have because now if the ``transactionType` is  not defined  we fallback to `TRANSACTION_REQUEST` but if it's a mobile agent that doesn't seem correct